content/learn/book/ecs/entities-components/_index.md

```diff
@@ -5,6 +5,68 @@ template = "book-section.html"
 page_template = "book-section.html"
 +++
 
-TODO: explain the basic data model
+As we discussed in the introduction to this chapter, **entities** represent objects in your game world, whose data is stored in the form of components.
 
-TODO: show how to create an entity, add components to it, and query for it in pure `bevy_ecs`
+The very first thing we're going to want to do is define the components that we'd like to use.
+To do so, we simply create Rust types (ensuring that they are `Send + Sync + 'static` so they can be sent across the threads safely) and pick a descriptive name for them.
+On this page, we're going to create a simple little combat system, so lets start by defining some basic components.
+
+```rust
+// These are dataless "unit structs", which hold no data of their own
+// In Bevy, these are useful for distinguishing similar entities or toggling behavior
+// and are called "marker components"
+struct Player;
+struct Enemy;
+
+// These simple components wrap an i8 in a tuple struct
+struct Life(u8);
+struct Attack(u8);
+struct Defense(u8);
+
+// We can store arbitrary data in our components, as long as it has a 'static lifetime
```

Comment:
People get confused all the time that types without lifetimes are `'static`, so I fear this comment is going to confuse those people.